tsa/README.md

```diff
@@ -98,7 +97,7 @@ ssh -p 2222 $TSA_HOST \
   < worker.json
 ```
 
-Note that in this case you should always have Garden and BaggageClaim listen on `127.0.0.1` so that they're not exposed to the outside world. For this reason there is no `$GARDEN_ADDR` or `$BAGGAGECLAIM_URL` as is the case with `register-worker`.
+Note that in this case you should always have Garden and BaggageClaim listen on `127.0.0.1` so that they're not exposed to the outside world. For this reason there is no `$GARDEN_ADDR` or `$BAGGAGECLAIM_URL`.
```

Comment:
Should we remove `$GARDEN_ADDR` and `$BAGGACLAIM_URL`  from the table as well?
And maybe this sentence below could be removed, because we do not  have `register-worker` any more.
```
For this reason there is no `$GARDEN_ADDR` or `$BAGGAGECLAIM_URL`.
```

What do you think?

Reply:
@pivotal-bin-ju
I think you are right, I remove it on the next commit